test/safe_math/safe_math.edl

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+
+  struct CountSizeParamStruct {
+    uint64_t count;
+    size_t size;
+    [count=count, size=size] uint64_t* ptr;
+  };
+
+  trusted {
+    public void enc_host_test();
+    public int enc_add_overflow([in, size=size] char* buf,
+                                size_t size);
+    public int enc_mul_overflow([in, count=count] int* buf,
+                                size_t count);
+    public int enc_deepcopy_in_overflow([in, count=1] CountSizeParamStruct* s);
+    public int enc_deepcopy_inout_overflow([out, count=1] CountSizeParamStruct* s);
+  };
+  untrusted {
+    int host_add_overflow([in, size=size] char* buf,
```

Comment:
What happens if both `count` and `size` are specified for a pointer parameter?

Reply:
We already emit error when both attributes are assigned to a pointer parameter.